backend/api/models/computetask.py

```diff
@@ -30,19 +32,17 @@ class ComputeTaskOutput(models.Model):
     permissions_process_public = models.BooleanField()
     permissions_process_authorized_ids = ArrayField(models.CharField(max_length=1024), size=100)
     transient = models.BooleanField(default=False)
+    channel = models.CharField(max_length=100)
 
     class Meta:
         unique_together = (("task", "identifier"),)
 
 
 class ComputeTaskOutputAsset(models.Model):
-    class Kind(models.TextChoices):
-        ASSET_MODEL = common_pb2.AssetKind.Name(common_pb2.ASSET_MODEL)
-        ASSET_PERFORMANCE = common_pb2.AssetKind.Name(common_pb2.ASSET_PERFORMANCE)
-
     task_output = models.ForeignKey("ComputeTaskOutput", on_delete=models.CASCADE, related_name="assets")
-    asset_kind = models.CharField(max_length=64, choices=Kind.choices)
+    asset_kind = models.CharField(max_length=64, choices=AlgoOutput.Kind.choices)
```

Comment:
Side improvement: we should use the algo output kind list here instead of redefining a list with the same values.

Reply:
(No change in migrations as the values are exactly the same by definition)